vue/components/ui/input-platforme/input-platforme.vue

```diff
@@ -32,19 +62,40 @@ input[type="text"] {
 }
 
 input[type="text"]:hover {
-    border-color: #dddddd;
+    background-color: #ebeef1;
+    border: solid 1px #dfe1e5;

```

Comment:
Isn't `border-color` enough? Also, use a named color instead of hex. Check: https://zpl.io/Vq56WpZ